webapp/src/components/meeting_settings/meeting_settings.jsx

```diff
@@ -122,7 +122,8 @@ export default class MeetingSettingsModal extends React.PureComponent {
                         <input
                             onInput={this.handleHashtagChange}
                             className='form-control'
-                            value={this.state.hashtag ? this.state.hashtag : ''}
+                            placeholder={this.state.hashtag ? this.state.hashtag : ''}
```

Comment:
Can we go the route of using `onChange` and `value`, rather than the prop set used here?

Reply:
Sorry, I didn't quite get it. I don't see the use of any props here. Everything is just local state variables.

Reply:
Sure, we can do that. Will pick this up in the coming weekend.

Reply:
@icy-meteor Just want to check if you got the time to get that change in? It's the last one before I can start cutting the next release!

Reply:
Sorry about delay. Not keeping well from quite some time, might be picking this up in the coming weekend.

Reply:
Oh, that is s bummer. Please take your time. I hope you feel better soon!

Reply:
@hanzei @mickmister @marianunez Have pushed the requested change :)